visual-tests/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "playwright",
```

Comment:
`playwright` is probably not the best name for this package.

Reply:
probably you are right. Can not confirm currently if it was my idea, or it was added automatically (`playright` is a name of our testing tool). Which name would be better?

Reply:
Something more descriptive, I think.
Currently, the internal packages are named:
- `handsontable-examples-internal`
- `handsontable-documentation`

So maybe something like `handsontable-visual-tests`? And while you're editing the file - it'd probably be a good idea to add a `"private": true,` entry to not publish the package by accident.

Reply:
sounds perfect. Pushed